### ISSUE TITLE
Capsule integration to consolev2 end2end tests

### ISSUE THREAD

**Issue by gordsport** [vegaprotocol/frontend-monorepo#467]

**Comment by dexturr** [comment]
Blocked by https://github.com/vegaprotocol/frontend-monorepo/issues/318

**Comment by dexturr** [comment]
Now blocked by being unable to have a market with some depth in capsule :+1:

**Comment by asiaznik** [comment]
On hold because of this: https://github.com/vegaprotocol/frontend-monorepo/issues/1310#issuecomment-1252444996
